pkg/juicefs/juicefs.go

```diff
@@ -544,17 +542,15 @@ func (j *juicefs) AuthFs(ctx context.Context, secrets map[string]string, setting
 		return cmd, nil
 	}
 
-	authCmd := j.Exec.CommandContext(ctx, config.CliPath, args...)
+	cmdCtx, cmdCancel := context.WithTimeout(ctx, 5*defaultCheckTimeout)
+	defer cmdCancel()
+	authCmd := j.Exec.CommandContext(cmdCtx, config.CliPath, args...)
 	envs := syscall.Environ()
 	for key, val := range setting.Envs {
 		envs = append(envs, fmt.Sprintf("%s=%s", key, val))
 	}
 	authCmd.SetEnv(envs)
-	var res []byte
-	_, err := util.DoWithinTime(ctx, 5*time.Second, nil, func() (err error) {
-		res, err = authCmd.CombinedOutput()
-		return
-	})
+	res, err := authCmd.CombinedOutput()
```

Comment:
If error is timeout, complete the return info? Like: "juicefs auth timeout..."

Reply:
Done